pkg/apis/cloudcredential/v1/aws_types.go

```diff
@@ -38,6 +38,8 @@ type StatementEntry struct {
 	Action []string `json:"action"`
 	// Resource specifies the object(s) this statement should apply to. (or "*" for all)
 	Resource string `json:"resource"`
+	// Condition specifies under which condition StatementEntry will apply
+	Condition IAMPolicyCondition `json:"condition,omitempty"`
```

Comment:
I'd prefer "iamPolicyCondition" or "policyCondition" to "condition" for the json key. Condition collides a little too much with typical status.conditions.

Reply:
Ok, fixing now.